skyportal/handlers/api/followup_request.py

```diff
@@ -1276,6 +1310,38 @@ async def get(self, instrument_id):
           description: |
             Arrow-parseable date string (e.g. 2020-01-01). If provided, end time
             of observation window, otherwise 12 hours from now.
+        - in: query
+          name: includeStandards
+          nullable: true
+          required: false
```

Comment:
Do we need nullable and required? I thought they were just opposite of each other...?

Reply:
good point.